docs/manage/_data.rst

```diff
@@ -0,0 +1,22 @@
+.. _manage_data_ref:
+
+Manage Data
+~~~~~~~~~~~
+
+.. note::
+  To manage Kolibri data you must sign-in as **Super user** or **Admin**.
+
+
+You can download Kolibri *Session* and *Summary* logs usage data and export in the CSV (comma separated values) format from the **Data** tab in your **Facility** dashboard.
+
+.. image:: img/export-usage-data.png
+  :alt: options for exporting usage data 
+
+
+Data Syncing
+------------
+
+``Coming soon!``
+
+
+.. Kolibri is capable to share your learner progress data with a central data repository when you are online. This enables the system to have an online backup of your data, allows you to view your student progress online, and to share your data across multiple Kolibri installations.
```

Comment:
Similar to above, it might be inappropriate to discuss our roadmap or timelines in the user docs.

Reply:
I'm fine with leaving all the `Coming soon` sections commented out for now.

Reply:
Changed to stay tuned